x-pack/plugin/esql/qa/testFixtures/src/main/java/org/elasticsearch/xpack/esql/CsvTestsDataLoader.java

```diff
@@ -300,7 +302,7 @@ private static void loadCsvData(
         }
```

Comment:
out-of-scope remark: hoo boy, the code in lines 197-302 is hard to read/skim. That would be great to factor out into a helper method some time in the future.

Reply:
yeh, agree.